src/devices/ControlBoardWrapper/ControlBoard_nws_ros.h

```diff
@@ -37,7 +37,7 @@
  * | Parameter name  | SubParameter   | Type    | Units          | Default Value | Required                    | Description                                                       | Notes |
  * |:---------------:|:--------------:|:-------:|:--------------:|:-------------:|:--------------------------: |:-----------------------------------------------------------------:|:-----:|
  * | node_name       |      -         | string  | -              |   -           | Yes                         | set the name for ROS node                                         | must start with a leading '/' |
- * | topic_name      |      -         | string  | -              |   -           | Yes                         | set the name for ROS topic                                        | must start with a leading '/' |
+ * | topic_name      |      -         | string  | -              |   -           | Yes                         | set the name for ROS topic                                        | must start with a leading '/', recommended value is /joint_states |
```

Comment:
I'm not sure if this is supposed to be `/joint_states` or `/<something>/joint_states`.